ui/src/app/components/html-viewer.tsx

```diff
@@ -42,25 +46,31 @@ export const HtmlViewer = withWindowSize()(
 
     private handleIframeLoaded() {
       try {
+        const { lastElementQuerySelector = MS_WORD_PARAGRAPH_CLASS } =
+          this.props;
         const iframeDocument = this.iframeRef.current.contentDocument;
         const { body } = iframeDocument;
         const openLinksInNewTab = iframeDocument.createElement('base');
-        const endOfPage = iframeDocument.createElement('div');
```

Comment:
This inserted an invisible element all the way to the left.  This is a problem because it can be off-screen if the user needs to scroll right to the document (up-down) scrollbars and the Proceed button.

Reply:
Ahhh, I didn't consider the intersection issue might be with the width...